R/quarto.R

```diff
@@ -337,3 +305,41 @@ quarto_binary_sitrep <- function(verbose = TRUE, debug = FALSE) {
 
   return(same_config)
 }
+
+
+wrap_quarto_error <- function(cnd, args, .call = rlang::caller_env()) {
+  if (!inherits(cnd, "system_command_status_error")) {
```

Comment:
I'd move this into the `withCallingHandlers` so that it's a separate `system_command_status_error = function(cnd) ...`

Reply:
Oh indeed. Thank you.